lib/applicationinsights.json

```diff
@@ -15,7 +20,18 @@
               "matchType": "regexp"
             }
           ],
-          "percentage": 1
+          "percentage": 0
+        },
+        {
+          "telemetryType": "request",
+          "attributes": [
+            {
+              "key": "code.namespace",
```

Comment:
Filters the Request logs from application insight that spams UploadLettersTask.run